virt-who.spec

```diff
@@ -87,47 +74,25 @@ install -m 644 virt-who-zsh %{buildroot}/%{_datadir}/zsh/site-functions/_virt-wh
 # registered to subscription-manager server
 
 %post
-%if %{use_systemd}
 %systemd_post virt-who.service
-%else
-# This adds the proper /etc/rc*.d links for the script
-/sbin/chkconfig --add virt-who
-%endif
 # This moves parameters from old config to remaining general config file
 %if (0%{?fedora} || 0%{?rhel} > 8)
 %{python_exec} %{python_sitelib}/virtwho/migrate/migrateconfiguration.py
 %endif
 
 %preun
-%if %{use_systemd}
 %systemd_preun virt-who.service
-%else
-if [ $1 -eq 0 ] ; then
-    /sbin/service virt-who stop >/dev/null 2>&1
-    /sbin/chkconfig --del virt-who
-fi
-%endif
 
 %postun
-%if %{use_systemd}
 %systemd_postun_with_restart virt-who.service
-%else
-if [ "$1" -ge "1" ] ; then
-    /sbin/service virt-who condrestart >/dev/null 2>&1 || :
-fi
-%endif
 
 
 %files
 %doc README.md LICENSE README.hyperv
 %{_bindir}/virt-who
 %{_bindir}/virt-who-password
 %{python_sitelib}/*
-%if %{use_systemd}
 %{_unitdir}/virt-who.service
-%else
-%{_sysconfdir}/rc.d/init.d/virt-who
```

Comment:
It would be also good to remove `virt-who-initscript` and installation step in `./setup.py` file.